sdks/python/test-suites/xlang/build.gradle

```diff
@@ -67,12 +67,22 @@ def ioXlang = new CrossLanguageTask().tap {
 
 // This list should include all expansion service targets in sdks/python/standard_expansion_services.yaml
 def servicesToGenerateFrom = [ioExpansionPath, gcpExpansionPath]
+def lowestSupportedVersion = project.properties['cross_language_validates_py_versions'].split(',')[0]
```

Comment:
let's add a variable, so that you don't have to parse xlang options. Example: https://github.com/apache/beam/blob/f08058ca75cf41cc535bab2e0f5cd6ef38c801ee/.github/actions/setup-default-test-properties/test-properties.json#L4 .

Reply:
This is referencing the property in this file (also used by other tests). Wouldn't it be better to keep it centralized? so we dont have to update it in multiple places
https://github.com/apache/beam/blob/f08058ca75cf41cc535bab2e0f5cd6ef38c801ee/sdks/python/test-suites/gradle.properties#L48

Reply:
> It goes through Py38's base_image_requirements file and uses the same Jinja dependency version. This way we're always installing the same version used to build the SDK.

Oh. how will jinja make its way in base_image_requirements? If it's not part of beam SDK dependencies, I think it won't be there. I suppose you cool look it up from `pyproject.toml`, or hardcode the same version in both places.

Reply:
Ahh I missed that part (thought pyproject.toml deps also show up in base_image_requirements)

I guess it's not worth the work to look it up and resolve for a version within the bounds? I'll opt for hardcoding it